src/datasources/fault-ds/partials/config.html

```diff
@@ -1,2 +1,44 @@
 <datasource-http-settings current="ctrl.current">
 </datasource-http-settings>
+
+<h3 class="page-heading">OpenNMS FM Datasource Details</h3>
+<div class="gf-form-group">
+    <div class="gf-form-inline">
+        <gf-form-switch class="gf-form"
+                        label="Use grafana user"
+                        tooltip="Defines if on supported alarm operations the currently logged in
+                                 Grafana user or the datasource user is used to talk to the ReST endpoint.
+                                 Supported operations are escalating, clearing, un-/acknowleding alarms as
+                                 well as creating/updating a journal/memo.
+                                 NOTE: Only OpenNMS admin users are allowed to perform actions as another user."
```

Comment:
I would rephrase this as:
NOTE: The data source must be configured using the 'admin' user in order to actions as other users.